Source/Core/DolphinQt/Debugger/CodeWidget.cpp

```diff
@@ -105,11 +105,16 @@ void CodeWidget::CreateWidgets()
   layout->setContentsMargins(2, 2, 2, 2);
   layout->setSpacing(0);
 
+  auto* top_layout = new QHBoxLayout;
   m_search_address = new QLineEdit;
   m_branch_watch = new QPushButton(tr("Branch Watch"));
-  m_code_view = new CodeViewWidget;
+  top_layout->addWidget(m_search_address);
+  top_layout->addWidget(m_branch_watch);
 
-  m_search_address->setPlaceholderText(tr("Search Address"));
```

Comment:
IMHO, the placeholder shouldn't be removed as it indicates what this field is used for.

Reply:
Oops, that was an accident.